static/script.js

```diff
@@ -223,3 +225,28 @@ function downloadURL(url, name) {
     document.body.removeChild(link);
     delete link;
 }
+
+function alertIfFileContainsPassword() {
+    passwords = []
+    let regex = /PASSWORD='.+'/i;
+
+    for (let child of getFiles(document.getElementById('file-container'))) {
+        let fileText = child.querySelector('#file-content-textarea').value;
+        if (checkPswd(fileText, regex)){
+            passwords.push(pswdsFound(fileText, regex));
+        }
+    }
+
+    if(passwords.length > 0){
+    let pswd_str = passwords.join(' ');
+        return confirm("You have have a password/s in your file. Consider removing it: " + pswd_str + "\n Continue anyway?");
```

Comment:
Actually maybe an short explanation on why this may be harmful would also be good. Something like `It will be sent to the server, do you want to continue anyway?`